app/script/entity/Conversation.js

```diff
@@ -176,30 +176,46 @@ z.entity.Conversation = class Conversation {
     this.hasActiveCall = ko.pureComputed(() => (this.hasLocalCall() ? this.call().isActiveState() : false));
     this.hasJoinableCall = ko.pureComputed(() => (this.hasLocalCall() ? this.call().canJoinState() : false));
 
-    this.unreadEvents = ko.pureComputed(() => {
+    this.unreadState = ko.pureComputed(() => {
       const unreadEvents = [];
       const messages = this.messages();
+      const unreadSelfMentions = [];
+      const unreadTextMessages = [];
```

Comment:
These are not necessarily test messages. The name is confusing. Files, images, videos would also fall in this category.